### DISCUSSION TITLE
SSL issue on the azuracast admin panel

### DISCUSSION THREAD

**Discussion by TheWatUpDude** [AzuraCast/AzuraCast#5651] [General]
I am not able to renew the SSL certificate on the azauracast admin panel. I tried to update it in the puty and digital ocean console but it says to update it in the azuracast admin panel. Please see the attached image [https://snipboard.io/USgLVz.jpg](url)

![lets encrypt azuracast error](https://user-images.githubusercontent.com/80156102/183266697-e93fccdb-14a7-4f32-b354-565d05ff5cea.png)

I need to get this resolved asap please. Because of this, this is not allowing the radio stream urls to work.

Help!

**Comment by ItsMitchh** [comment]
Fixed in discord. User needed to update his base site URL to include HTTP://
